dvc/ignore.py

```diff
@@ -33,10 +35,16 @@ def __init__(self, pattern_list, dirname, sep):
         self.pattern_list = pattern_list
         self.dirname = dirname
 
-        self.regex_pattern_list = [
-            GitWildMatchPattern.pattern_to_regex(pattern_info.patterns)
-            for pattern_info in pattern_list
-        ]
+        self.regex_pattern_list = []
```

Comment:
Looks good. Do you have benchmarks for this?

Reply:
No, I hadn't, The PR only did some name field jobs and didn't touch the matching logic.